locales/it/index.yml

```diff
@@ -886,10 +885,19 @@ wallet:
           title: "Non riusciamo a contattare la tua banca"
           body: "Siamo al lavoro per risolvere il problema."
       search:
+        loading: "Stiamo recuperando la lista delle tue carte\n\nAttendi qualche secondo"
         koTimeout:
-          title: "TMP KO Timeout add"
+          title: "Non riusciamo a recuperare le tue carte"
+          body: "Riprova per favore"
         koNotFound:
-          title: "TMP KO Timeout add"
+          title: "Spiacenti, non abbiamo trovato nessuna carta tra le banche aderenti al servizio"
+          body: "Non risultano carte con circuito internazionale intestate a te: contatta la tua Banca per ulteriori informazioni."
+        koSingleBankNotFound:
+          title: "Spiacenti, non abbiamo trovato nessuna carta su questa banca"
+          body: "Non risultano carte con circuito internazionale intestate a te. Se lo desideri, IO può effettuare una ricerca tra tutte le banche aderenti."
+        koServiceError:
+          title: "Non riusciamo a recuperare le tue carte"
```

Comment:
Maybe the design has been updated, btw it looks different from https://www.pivotaltracker.com/story/show/176576695/comments/221296911

Reply:
In this phase we are going to use the MVP screen: https://pagopa.invisionapp.com/console/Aggiunta-Carte-Co-Badge-ckk5og2dh05a9011v6s8edalw/ckki6dlwu01at01129sqr2muq/play